extensions/schema-compare/tsconfig.json

```diff
@@ -1,8 +1,7 @@
 {
+	"extends": "../shared.tsconfig.json",
```

Comment:
extensions/dacpac too please...

Reply:
That's already using the shared config file